api/openapi.yaml

```diff
@@ -2277,6 +2331,11 @@ components:
             - aarch64
             - x86_64
           type: string
+        metadata:
+          description: An object of key/value associated with an Image
+          type: object
+          properties:
+            $ref: '#/components/schemas/ImageMetadataAnnotationKeyValuePair'
```

Comment:
This is the wrong object here. The 'ImageMetadataAnnotationKeyValuePair' includes the 'operation' value, which is not part of the Image metadata object. This needs to be a list of <string,string> pairs.